### DISCUSSION TITLE
Ability to enter sale prices for individual products

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/commerce#2295] [Ideas]
> Created by: Cole Henley ([cole@ournameismud.co.uk](cole@ournameismud.co.uk)) on 2016/12/06 16:23:44 +0000
Votes at time of UserVoice import: 8

------
Ability within Product entry page to apply a sale price specifically for that project. So ideally some kind of inline sale price definition would be ace!

**Comment by lukeholder** [comment]
Variant specific sales are a good idea, run before the general sales.

**Comment by russback** [comment]
Keen on this one as discounting at an item level is pretty standard on other ecomm platforms and currently you need to set up a sale that assumes the same discount policy across variants.

**Comment by alexanderloewe** [comment]
Please keep up the pressure on this issue. This is a very basic feature that is standard on other commerce software or platforms.

**Comment by engram-design** [comment]
I was thinking about how to address this for a client, where they wanted to have a rule for "Buy 6+ of any item and get them for $25". I suppose it's more of a discount, instead of a sale.

Right now, you can only do a percentage off, or a flat dollar amount off each item. As each item can vary in price, it certainly won't work to have a dollar amount off, and percentage is almost there, but not what the client wants.

For example, you might have the following cart contents:

- Item 1: $50
- Item 2: $60
- Item 3: $65
- Item 4: $45
- Item 5: $30
- Item 6: $25

And if I wanted to apply the above discount rule, every item would become $25.

So I suppose either an amendment to the discounts logic where you set a specific line item price in the discount settings for each matching item, or setting a sale/discount price of sort in the product itself. I suppose the downside of the latter is what if you want different prices based on different discounts/sales?
